package.json

```diff
@@ -38,7 +38,9 @@
     "publish": "lerna publish",
     "test": "./scripts/test.js",
     "test-latest-cra": "npm --prefix lib/cli run test-latest-cra",
-    "repo-dirty-check": "node ./scripts/repo-dirty-check"
+    "repo-dirty-check": "node ./scripts/repo-dirty-check",
+    "storybook": "start-storybook -p 6006",
+    "chromatic": "chromatic test --storybook-addon --app-code sm3026v658i"
```

Comment:
If it's not a public key, probably better hide it with the CI parameter

Reply:
Yeah, see the note on the commit ;)

Reply:
Hah, 😄